ui/package.json

```diff
@@ -23,12 +23,14 @@
     "axios": "^0.28.1",
     "bootstrap": "^5.3.2",
     "bootstrap-icons": "^1.10.5",
+    "buffer": "^6.0.3",
     "classnames": "^2.3.1",
     "codemirror": "^6.0.1",
     "color": "^4.2.3",
     "copy-to-clipboard": "^3.3.2",
     "dayjs": "^1.11.5",
     "diff": "^5.1.0",
+    "gray-matter": "^4.0.3",
     "i18next": "^21.9.0",
```

Comment:
I see that you use gray-matter (38.6 kB) to handle the front matter content, but this dependency must be used in conjunction with buffer (91.3 kB), the size of these two dependencies is relatively large, which will affect the loading speed of the page, it is recommended that you can use some lightweight libraries such as front-matter (11.3 kB), which will greatly reduce the size of the build. This will greatly reduce the size of the build.

Reply:
thanks for pointing that out. replaced with `front-matter` and removed `gray-matter` and `butter`